blog/src/components/Featured/CardWide.jsx

```diff
@@ -0,0 +1,54 @@
+import images from "../../utils/images";
+import "../../styles/Card.scss";
+import PropTypes from "prop-types";
+
+
+const CardWide = ({ blogItemData }) => {
```

Comment:
It would be better to add some drop shadow or border to make the card identifiable.

![image](https://github.com/FaizaanFazal/Blog-Website-Using-Test-Driven-Development/assets/35307862/3403e939-ba7e-44e5-b3f4-8af0f3aa897d)